src/JustSaying/Messaging/MessageSerialization/SystemTextJsonMessageBodySerializer.cs

```diff
@@ -18,7 +15,7 @@ public abstract class SystemTextJsonMessageBodySerializerBase
     /// <item><description>Using a <see cref="JsonStringEnumConverter"/> for enum serialization.</description></item>
     /// </list>
     /// </remarks>
-    private protected static readonly JsonSerializerOptions DefaultJsonSerializerOptions = new()
+    public static readonly JsonSerializerOptions DefaultJsonSerializerOptions = new()
```

Comment:
As this is public now, should probably be a property?

Reply:
whoops, that was the intention 🤦